calico-cloud/threat/container-threat-detection.mdx

```diff
@@ -85,11 +89,58 @@ EOF
 
 ### Monitor the Security Events page for malicious programs
 
-If a malicious or suspicious program is run within the cluster, it will be reported on the Security Events page of the
+If a malicious or suspicious program is run within the cluster, it will be reported on the **Security Events** page of the
 {{prodname}} UI.
 
 ![malware-alert-example](/img/calico-cloud/malware-alert-example.png)
 
+### Update detectors settings
+
+Threat detection capabilities are provided by detectors that protect against specific [MITRE techniques](https://attack.mitre.org/techniques/enterprise/). 
+Detectors are grouped by [MITRE tactics](https://attack.mitre.org/tactics/enterprise/) they protect against. Additional information for each detector can be found by clicking on a detector. 
+All detectors are enabled by default, with the option of disabling specific detectors if needed.
+
+![CTD Detector Settings](/img/calico-cloud/container-threat-detection-detector-settings.png)
+
+#### Configure detectors via RuntimeSecurity Custom Resource
+
+Detector settings configured through the UI are stored in the cluster's RuntimeSecurity Custom Resource. When configuring multiple clusters, the same configuration can be applied to all clusters.
+
+Get the reference `detectorsConfig`:
+
+```
+export detectorConfig="$(kubectl get runtimesecurities.operator.tigera.io default -o jsonpath="{.spec.detectorsConfig}")"
+```
+
+Alternatively, can enable Container Threat Detection with the required configuration in one command:
```

Comment:
I don't think "Alternatively" is correct here.  The next line is the second step of the process, not an alternative.

Also, don't need the `export`.  (And it looks odd to use it when not needed.)